level-2/예상-대진표.js

```diff
@@ -13,3 +13,30 @@ function solution(n, a, b) {
     }
     return currentRound
 }
+
+//정답 2 - le2sky
+function solution(n, a, b) {
+  let arr = Array.from({ length: n }, () => 0)
+  arr[b - 1] = "B"
+  arr[a - 1] = "A"
+
+  const isDiff = () => {
+    return (
+      (arr.indexOf("A") + 1 > arr.length / 2 && arr.indexOf("B") + 1 <= arr.length / 2) ||
+       (arr.indexOf("A") + 1 <= arr.length / 2 && arr.indexOf("B") + 1 > arr.length / 2)) ? true : false
+  };
+  const isLeft = () => {
+    return (arr.indexOf("A") + 1 > arr.length / 2) ? false : true
```

Comment:
🌱 반영해도 좋고 넘어가도 좋아요. 🌱

이전 코멘트와 동일하게 불필요한 boolean 값들을 제거하시면 더 좋을 것이라 생각합니다!
```suggestion
    return arr.indexOf("A") + 1 <=  arr.length / 2
```

Reply:
이 생각은 전혀 못했었네요 항상 좋은 리뷰 감사드립니다! ㅎㅎ